includes/payment-methods/class-wc-gateway-stripe-multibanco.php

```diff
@@ -0,0 +1,434 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class that handles Multibanco payment method.
+ *
+ * @extends WC_Gateway_Stripe
+ *
+ * @since 4.0.0
```

Comment:
Change all versions to 4.1.0

Reply:
@roykho only @version or also @since ?

Reply:
both.

Reply:
@roykho thanks. that is except for the Get_icon function where `@since` should stay 1.0.0, correct?

Reply:
Yeah so just mainly the 4.0.0 ones need to change.